index.html

```diff
@@ -35,184 +32,163 @@
-                            feature or website. Look through some of my work
-                            and experience! If you like what you see and
-                            have a project you need coded,
-                            don’t hestiate to contact me.</p>
-                        <ul class="social">
-                            <li><a href="https://twitter.com/LukmanAbdulka18"><img src="images\twitter.svg"
-                                        alt="twitter"></a></li>
-                            <li><a href="#"><img src="images\Linkedin.svg" alt="linkedin"></a></li>
-                            <li><a href="#"><img src="images\medium.svg" alt="medium"></a></li>
-                            <li><a href="https://github.com/lukman155"><img src="images\github.svg" alt="github"></a>
-                            </li>
-                            <li><a href="#"><img src="images\angellist.svg"></a></li>
-                        </ul>
-                    </div>
+    <!--Hero/Intro Page-->
 
+    <main class="background flex-container">
+        <ul class="header-desktop">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        <h1 class='h1-top-space'>Hey there. I’m Lukman</h1>
+        <h1 class="h1-highlight">I’m a software developer</h1>
+            
+        <p class="hero-text">I can help you build a product,
+            feature or website. Look through some of my work
+            and experience! If you like what you see and
+            have a project you need coded,
+            don’t hestiate to contact me.</p>
+
+        <ul class="social-icons">
+            <li><a href="https://twitter.com/LukmanAbdulka18"><img src="images\twitter.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images\Linkedin.svg" alt="linkedin"></a></li>
+            <li><a href="#"><img src="images\medium.svg" alt="medium"></a></li>
+            <li><a href="https://github.com/lukman155"><img src="images\github.svg" alt="github"></a></li>
+            <li><a href="#"><img src="images\angellist.svg"></a></li>
+        </ul>
+    </main>
+
+    <!--Project Section-->
+
+    <section class="flex-container">
+        <h2>My Recent Works</h2>
+        <hr>
+        <article class="grid-container">
+            <div class="project-frame flex-container">
+                <img class="project-image" src="/images/project-img.png" alt="project desc">
+                <div class="project-detail flex-container">
+                    <h2 class="project-name">Project Name</h2>
+                    <ul class="skills">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button type="button" class="projectbutton">See Project</button>
```

Comment:
- [ ] the see project button should be a link that takes you to an external link or project etc .. so kindly wrap all of your `See Project` buttons inside an anchor tag

![image](https://user-images.githubusercontent.com/89282221/193098130-de4fcb29-a0f8-4215-973b-12a82e221513.png)